.gitignore

```diff
@@ -164,3 +164,8 @@ cython_debug/
 
 # test data directory
 data
+
+# local manifests
+src/polus/plugins/_plugins/manifests/*
```

Comment:
I would suggest not using the source directory to stored manifests, unless I am missing some implementation details here.

Reply:
This is necessary for when a developer installs `polus-plugins` with `pip install -e .`